docs/install/debug.md

```diff
@@ -177,6 +177,15 @@ use `nrn-enable-sanitizer special -python path/to/script.py` or
 because the `python` binary is (presumably) not linked against the sanitizer
 runtime library.
 
+If ctest or nrn-enable-sanitizer runs generate a sanitizer error like
+`AddressSanitizer: CHECK failed: asan_interceptors.cpp`
+it might be that the automatically determined `LD_PRELOAD` is insufficient.
+(It happened to me with ubuntu 24.04 + gcc 13.2.0). In this case you
+can temporarily set the `OVERRIDE_LD_PRELOAD` environment variable before
```

Comment:
@nrnhines : just one very minor question / suggestion: should we add **_NRN__** prefix to the env variable name so that it’s according to convention and may not mix with others?

Reply:
I will do that. An alternative, instead of environment variable is a cmake variable like ```-DNRN_OVERRIDE_LD_PRELOAD="complicated shell expression"```. However, my expectation is that the need for a user setable LD_PRELOAD alternative instead of cmake derived default for the sanitizer will become obsolete when a newer version of the sanitizer (or whatever is the underlying reason for the problem) becomes available for ubuntu.